docs/install/installEverest.md

```diff
@@ -29,7 +29,134 @@ Percona Everest assists with installing all the necessary operators and required
 
 We recommend setting up Percona Everest on the [Amazon Elastic Kubernetes Service (EKS)](../quickstart-guide/eks.md) or on [Google Kubernetes Engine (GKE)](../quickstart-guide/gke.md), as Percona Everest may not work as expected on local Kubernetes installations (minikube, kind, k3d, or similar products) due to network issues.
 
-## Prerequisites
+## Install to Kubernetes (recommended)
+
+### Prerequisites
+
+Before getting started with Percona Everest, we recommend that you:
+{.power-number}
+
+1. Install [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html){:target="_blank"} for EKS or the [gcloud CLI](https://cloud.google.com/sdk/docs/install){:target="_blank"} for GKE.
+2. Verify that you have access to the Kubernetes cluster that you want to use with Everest. By default, Everest uses the kubeconfig file available under *~/.kube/config*. Run the following command:
+    
+    ```sh 
+    kubectl get nodes
+    ```
+
+    ??? example "Expected output"
+        ```
+            NAME                                       STATUS   ROLES    AGE   VERSION
+            gke-<name>-default-pool-75d48bfc-bx8g      Ready    <none>   11h   v1.26.7-gke.500
+            gke-<name>-default-pool-75d48bfc-c2df      Ready    <none>   11h   v1.26.7-gke.500
+            gke-<name>-default-pool-75d48bfc-zl7k      Ready    <none>   11h   v1.26.7-gke.500
+        ```
+
+### Install and provision
+
+To install and provision Percona Everest to Kubernetes:
+{.power-number}
+
+1. Create the `percona-everest` namespace:
+    ```sh
+    kubectl create namespace percona-everest
+    ```
+2. Deploy Everest to Kubernetes:
+    ```sh
+    kubectl apply -f https://raw.githubusercontent.com/percona/percona-everest-backend/v0.4.0/deploy/quickstart-k8s.yaml -n percona-everest
+    ```
+3. (Optional) Verify that the services started correctly:
+    
+    ```sh
+    kubectl get pods -n percona-everest
+    ```        
+    ??? example "Expected output"
+        ```
+        NAME                                                   READY   STATUS    RESTARTS       AGE
+        percona-everest-0                                      2/2     Running   2 (10s ago)    10s
+        ```
+
+4. Make a note of the external IP address for the Everest service (for instance, 127.0.0.1 in this example):
+   
+    ```sh 
+    kubectl get svc/everest -n percona-everest
+    ```
+    
+    ??? example "Expected output"
+        ```
+        NAME      TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)          AGE
+        everest   LoadBalancer   10.43.172.194   127.0.0.1       8080:31611/TCP   10s
+        ```
+
+5. Rename the downloaded file using the following command and replacing the placeholder `everestctl-darwin-amd64` to match the file downloaded in the previous step:
```

Comment:
I missed something when I suggested these steps :facepalm: 
We are missing this step between step 4 and 5.
```
4. Download the latest release of [everestctl](https://github.com/percona/percona-everest-cli/releases){:target="_blank"} to provision Percona Everest.```